requirements.txt

```diff
@@ -0,0 +1,2 @@
+feast_custom_provider/
```

Comment:
This is cool, didn't know we could specify subdirs in the requirements.txt file.